drake/CMakeLists.txt

```diff
@@ -203,7 +214,7 @@ else()
    endif()
 endif()
 
-include_directories(thirdParty/spruce/include)
+include_directories(thirdParty/spruce/include) # FIXME: should be in build/include
```

Comment:
i don't agree with the fixme.  i think we should only reference prerequisite externals in build/include, not anything contained inside of drake.